### PR TITLE
Add test cases for field update operators

### PR DESCRIPTION
**Pull request by noisersup**
Closes #39.

### CODE REVIEW

**Review by chilagrow — COMMENTED** [review]
Could you make the PR smaller? It's quite difficult to follow so many operators. Same for other PRs.

**Comment by noisersup** [comment]
@chilagrow I group operators by their documentation groups. Even though there's 41 files changed here, most of them are generated, so there's no need to check most of them (js files).
This PR introduces 8 tests for operators (in json files), which test for a basic logic. Does it need a split?

**Comment by chilagrow** [comment]
> @chilagrow I group operators by their documentation groups. Even though there's 41 files changed here, most of them are generated, so there's no need to check most of them (js files). This PR introduces 8 tests for operators (in json files), which test for a basic logic. Does it need a split?

@noisersup thanks, I won't look at `js` files.

File: cts/update/field/max.json [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+      "update": "values",
+      "updates": [
+        {
+          "q": {
+            "_id": "int32"
+          },
+          "u": {
+            "$max": {
+              "v": {
+                "$numberInt": "50"
```
Comment (chilagrow) [inline cts/update/field/max.json:13]: In CTS, how do we plan to test different test cases such as `{$max: {v: 20}}` and v is left as 42? Possibly we are currently testing each simple case?

Comment (noisersup) [reply cts/update/field/max.json:13]: Yes, currently we test for the simplest behavior, just to cover all operators. Later, we're going to test each field, and corner cases.

**Review by chilagrow — APPROVED** [review]
